### DISCUSSION TITLE
Welcome to yt-channel-downloader Discussions!

### DISCUSSION THREAD

**Discussion by hyperfield** [hyperfield/yt-channel-downloader#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Zeovnne** [comment]
Hello, I'm stuck at installing YT channel downloader and I don't know anything about programming languages. Would you mind showing the steps by posting photos or videos? Thank you!
